core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthChecker.java

```diff
@@ -174,6 +174,10 @@ public void onSubscribe(Subscription subscription) {
 
         @Override
         public void onNext(HttpObject obj) {
+            // There's no chance that an exception is raised from the underlying logic, so we don't do
```

Comment:
These comments seem very noisy. Do we need them? I feel like just leaving it to a `SafeSubscriber` abstract class that always adds try / catch is better than adding these comments that reduce the readability of the code.